src/components/banner/banner.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./banner.scss";
+
+export const Banner = ({ Color, Title, Link }) => {
+  return (
+    <div className={`banner-wrapper is-${Color}`}>
```

Comment:
When color is not defined, we will end up with `banner-wrapper is-` as CSS classes. What we can do:
1. Add a default value to `color` like [here](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values) or we add to the propType `isRequired`